dbt/dbt_project.yml

```diff
@@ -35,6 +35,16 @@ vars:
   # but can also be use to select specific time frames for testing
   data_test_iasworld_year_end: 2030
 
+  # Latest year of iasWorld data available. Necessary since iasWorld does
+  # not rollover into the new year on January 1st, so the latest ias year lags
+  # the calendar year by a small amount. Use this instead of YEAR(CURRENT_DATE)
+  # if you want to query for the latest year of iasWorld data.
+  #
+  # WARNING: Be sure to rebuild all models that depend on this variable when
+  # you bump its value, since otherwise they will be stuck querying the prior
+  # year of data
```

Comment:
This is an annoying limitation, but I expect it not to affect us in practice, since [including variable changes in the `state:modified` selector is queued up for dbt v1.10](https://github.com/dbt-labs/docs.getdbt.com/pull/6222#issuecomment-2484025388). If that feature lands on time, it means that the next time we need to bump this variable in 2026, dbt will automatically rebuild models that depend on it during the associated `build-and-test-dbt` CI run. If v1.10 does indeed include this feature, I'll plan to remove this warning during the upgrade PR.